### ISSUE TITLE
[bug] project timeline cannot read property type undefined

### ISSUE THREAD

**Issue by deantheiceman** [OCA/project#672]
I can't seem to get project timeline to work in version 12, which fields are required to be put into the tasks?

Sometimes when navigating to project timeline page we get this error. Or when we double click on the timeline

```
Error:
Uncaught TypeError: Cannot read property 'type' of undefined

https://odoo/web/content/1595-790dd3a/web.assets_backend.js:2574
Traceback:
TypeError: Cannot read property 'type' of undefined
    at Class.select_field (https://odoo/web/content/1595-790dd3a/web.assets_backend.js:2574:397)
    at Class.select_field (https://odoo/web/content/1594-554094b/web.assets_common.js:3541:371)
    at Class.changed (https://odoo/web/content/1595-790dd3a/web.assets_backend.js:936:1284)
    at HTMLSelectElement.<anonymous> (https://odoo/web/content/1594-554094b/web.assets_common.js:3643:11)
    at HTMLDivElement.dispatch (https://odoo/web/content/1594-554094b/web.assets_common.js:892:451)
    at HTMLDivElement.elemData.handle (https://odoo/web/content/1594-554094b/web.assets_common.js:865:173)
    at Object.trigger (https://odoo/web/content/1594-554094b/web.assets_common.js:886:211)
    at HTMLSelectElement.<anonymous> (https://odoo/web/content/1594-554094b/web.assets_common.js:937:147)
    at Function.each (https://odoo/web/content/1594-554094b/web.assets_common.js:625:767)
    at jQuery.fn.init.each (https://odoo/web/content/1594-554094b/web.assets_common.js:613:643)
```

**Comment by clxadmin** [comment]
I have the same issue here with Odoo 13 CE

TypeError: Cannot read property 'default_group_by' of undefined
TypeError: Cannot read property 'default_group_by' of undefined
    at Class.init (https://odoo/web/content/342-d2be89a/web.assets_backend.js:4315:751)
    at Class.prototype.<computed> [as init] (https://odoo/web/content/335-a2c9999/web.assets_common.js:3861:488)
    at new Class (https://odoo/web/content/335-a2c9999/web.assets_common.js:3862:65)
    at Class._createViewController (https://odoo/web/content/342-d2be89a/web.assets_backend.js:441:658)
    at newController (https://odoo/web/content/342-d2be89a/web.assets_backend.js:454:107)
    at Class._switchController (https://odoo/web/content/342-d2be89a/web.assets_backend.js:454:751)
    at Class._onSwitchView (https://odoo/web/content/342-d2be89a/web.assets_backend.js:462:6)
    at Class.<anonymous> (https://odoo/web/content/335-a2c9999/web.assets_common.js:3945:11)
    at Class.trigger (https://odoo/web/content/335-a2c9999/web.assets_common.js:3943:180)
    at Class._trigger_up (https://odoo/web/content/335-a2c9999/web.assets_common.js:3949:378)

Steps to Reproduce:
1) Install the module Version version": "13.0.1.0.0" to a fresh installed Odoo 13 CE, refresh Odoo page
2) Go to Project Dashboard - click on icon 'view timeline' -- error appears
Go to Tasks (All tasks or Project >> Task) and click on icon 'view timeline' -- error appears

**Comment by pedrobaeza** [comment]
Please indicate exact steps and data to reproduce the problem.

**Comment by deantheiceman** [comment]
Same steps to preproduce:

1. Install the module on fresh Odoo 12 CE, refresh odoo page
2. Go to projects dashboard and click on view timeline, error shows

Sometimes the error also doesn't show but when I double click on the interface the error shows up

**Comment by pedrobaeza** [comment]
Are you reproducing the problem with the "try me on runbot" button? The problem is the data that makes this to fail, so please indicate this.

**Comment by JordiADARIA** [comment]
Hi, do you have installed "field service" module? that module makes project timeline fails, as on issue:


https://github.com/OCA/field-service/issues/379

It seems field service changes something in timeline that makes that project timeline crashes. Tested on runbot, and on fresh db installations.

Uninstalling field-service module, its running perfectly.

**Comment by brian10048** [comment]
This can be closed
Fixed in OCA/field-service#739
